packages/macros/src/index.ts

```diff
@@ -46,6 +46,8 @@ import VueShortEmits from '@vue-macros/short-emits'
 import VueShortVmodel from '@vue-macros/short-vmodel'
 import VueDefineOptions from 'unplugin-vue-define-options'
 
+export { shallowRef as useRef } from 'vue'
```

Comment:
Why we export alias here?

Reply:
use `import { useRef } from 'unplugin-vue-macros'` instead of  
`import { shallowRef as useRef } from 'vue'`.

Reply:
Let's move the export to `unplugin-vue-macros/runtime` since the main export is for bundlers and may include code that can't run in browsers if treeshaking fails.